src/game/client/gui/regiontitle.lua

```diff
@@ -12,9 +12,9 @@ function regiontitle:regiontitle( parent, name )
 	self.height = 72
 	self:setPos( 0, graphics.getViewportHeight() / 2 - self.height / 2 )
 
-	local titleFont  = "fonts/Mark Simonson - Proxima Nova Bold.otf"
+	local titleFont  = "fonts/SourceSansPro-Bold.otf"
```

Comment:
On hold, I think this is appropriate, but it's a `vertex-adventure` specific HUD element, which uses these licensed fonts